### ISSUE TITLE
Cannot read property 'startsWith' of undefined

### ISSUE THREAD

**Issue by Gert-dev** [twifty/atom-phpunit-integrator#46]
Encountering this one after the update to 0.0.51:

```
TypeError: Cannot read property 'startsWith' of undefined
    at /home/werk/.atom/packages/shared-settings/lib/shared-settings.js:18:67
    at PaneContainer.observePaneItems (/usr/lib/atom/src/pane-container.js:104:45)
    at WorkspaceCenter.observePaneItems (/usr/lib/atom/src/workspace-center.js:71:59)
    at CompositeDisposable.getPaneContainers.map.container (/usr/lib/atom/src/workspace.js:637:62)
    at Array.map (native)
    at Workspace.observePaneItems (/usr/lib/atom/src/workspace.js:637:35)
    at Object.activate (/home/werk/.atom/packages/shared-settings/lib/shared-settings.js:17:47)
    at Package.activateNow (/usr/lib/atom/src/package.js:225:27)
    at measure (/usr/lib/atom/src/package.js:201:27)
    at Package.measure (/usr/lib/atom/src/package.js:83:19)
    at activationPromise.Promise (/usr/lib/atom/src/package.js:195:14)
    at Promise (<anonymous>)
    at Package.activate (/usr/lib/atom/src/package.js:193:32)
    at PackageManager.activatePackage (/usr/lib/atom/src/package-manager.js:684:36)
    at config.transactAsync (/usr/lib/atom/src/package-manager.js:659:30)
    at Config.transactAsync (/usr/lib/atom/src/config.js:866:22)
    at PackageManager.activatePackages (/usr/lib/atom/src/package-manager.js:657:17)
    at PackageManager.activate (/usr/lib/atom/src/package-manager.js:636:44)
    at loadStatePromise.loadState.then (/usr/lib/atom/src/atom-environment.js:857:21)
    at <anonymous>
```

I was also encountering a problem related to an unknown setting `alwaysOpen` in 0.0.50, but I see you've modified code related to that in 0.0.51, so it's probably fixed (if it isn't, I'll let you know).

**Comment by twifty** [comment]
A few teething bugs in the new `shared-settings` package.

The `alwaysOpen` was fixed in 0.0.51. I renamed all settings to their kebab-case variants. That was one I missed.

**Comment by twifty** [comment]
Fixed in shared-settings@0.0.5

One of the items passed to `atom.workspace.observePaneItems` has a 'getURI' method, but returns undefined. All I can do is check if a string is returned.
